### PR TITLE
If community plugin has the `vipgo-helper.php` file, load it

### PR DESCRIPTION
**Pull request by ethitter**
Community plugins could include our helpers, such as with `msm-sitemap`.

### CODE REVIEW

**Comment by ethitter** [comment]
This might belong in one of the UI classes instead, but this is a start.

See also https://github.com/Automattic/vip-go-mu-plugins/issues/580.

File: vip-dashboard.php [hunk 1]
```diff
@@ -346,3 +346,25 @@ function wpcom_vip_menu_order( $menu_ord ) {
...
 	return $vip_order;
 }
+
+/**
+ * Load `vipgo-helper.php` if it exists for a community plugin
+ *
+ * Loaded at priority 6 because all plugins are typically loaded before 'plugins_loaded', and the UI-enabled plugins use priority 5
+ */
+function wpcom_vip_load_helpers_for_community_plugins() {
+	$community_plugins = get_option( 'active_plugins' );
```
Comment (mjangda) [inline vip-dashboard.php]: Should we use `wp_get_active_and_valid_plugins()` instead?

File: vip-dashboard.php [hunk 2]
```diff
@@ -346,3 +346,25 @@ function wpcom_vip_menu_order( $menu_ord ) {
 
 	return $vip_order;
 }
+
+/**
+ * Load `vipgo-helper.php` if it exists for a community plugin
+ *
+ * Loaded at priority 6 because all plugins are typically loaded before 'plugins_loaded', and the UI-enabled plugins use priority 5
```
Comment (mjangda) [inline vip-dashboard.php]: We should move this inline with the `add_action` call.

**Comment by ethitter** [comment]
> Do we need to care about network-activated plugins too?

Good catch!

Also, on further review, this actually belongs in the mu-plugins, not the dashboard, so I'm closing this in favor of https://github.com/Automattic/vip-go-mu-plugins/issues/580.

**Pull request closed without merging**